views/layouts/docs.hbs

```diff
@@ -45,25 +45,58 @@
 </div>
 
 <div class='py-6 py-md-7 py-lg-8'>
-  <div class='container-lg p-responsive docs strong-headers'>
-    <div class="markdown-body">{{{body}}}</div>
-    <hr>
+  <div class='full-width p-responsive docs strong-headers'>
+      <div class="col-sm-2 float-left p-4">
+      {{#if false}}
+        <span />
+      {{/if}}
+      </div>
+
+    <div class="col-md-8 float-left p-4">
+      <div class="markdown-body">{{{body}}}</div>
+      <hr>
+
+      <div class="doc-actions text-center">
+        <a class="mr-4 propose-change" href='{{doc.githubUrl}}'>
+          <span class="octicon octicon-mark-github"></span>
+          {{localized.docs.footer.propose_change}}
+        </a>
 
-    <div class="doc-actions text-center">
-      <a class="mr-4 propose-change" href='{{doc.githubUrl}}'>
-        <span class="octicon octicon-mark-github"></span>
-        {{localized.docs.footer.propose_change}}
-      </a>
+        <a class="mr-4 translate-on-crowdin" href='{{doc.crowdinUrl}}'>
+          <span class="octicon octicon-globe"></span>
+          {{localized.docs.footer.translate}}
+        </a>
 
-      <a class="mr-4 translate-on-crowdin" href='{{doc.crowdinUrl}}'>
-        <span class="octicon octicon-globe"></span>
-        {{localized.docs.footer.translate}}
-      </a>
+        <a class="mr-4" href='{{doc.href}}/history'>
+          <span class="octicon octicon-history"></span>
+          {{localized.docs.footer.version_history}}
+        </a>
+      </div>
+    </div>
+
+    <div class="col-sm-2 float-left p-4">
+      {{#if false}}
```

Comment:
Are these `#if false` clauses just to disable the inner elements for now?

Reply:
Yes, outer element still be visible because they needed to create the grid.